src/erlcloud_securityhub.erl

```diff
@@ -0,0 +1,129 @@
+-module(erlcloud_securityhub).
+
+-include("erlcloud.hrl").
+-include("erlcloud_aws.hrl").
+
+%% API
+-export([
+    describe_hub/1,
+    describe_hub/2
+]).
+
+-type securityhub() :: proplist().
+
+-type param_name() :: binary() | string() | atom().
+-type param_value() :: binary() | string() | atom() | integer().
+-type params() :: [param_name() | {param_name(), param_value()}].
+
+-spec describe_hub(AwsConfig) -> Result
+    when AwsConfig :: aws_config(),
+    Result :: {ok, securityhub()} | {error, not_found} | {error, term()}.
+describe_hub(AwsConfig)
+    when is_record(AwsConfig, aws_config) ->
+    describe_hub(AwsConfig, _Params = []);
+describe_hub(Params) ->
+    AwsConfig = erlcloud_aws:default_config(),
+    describe_hub(AwsConfig, Params).
+
+-spec describe_hub(AwsConfig, Params) -> Result
+    when AwsConfig :: aws_config(),
+    Params :: params(),
+    Result :: {ok, securityhub()}| {error, not_found} | {error, term()}.
+describe_hub(AwsConfig, Params) ->
+    Path = ["accounts"],
+    case request(AwsConfig, _Method = get, Path, Params) of
+        {ok, Response} ->
+            {ok, Response};
+        {error, {<<"ResourceNotFoundException">>, _Message}} ->
+            {error, not_found};
+        {error, {<<"InvalidAccessException">>, _Message}} ->
```

Comment:
I would not wrap this one. let it fall through into `{error, Reason}`

Reply:
done.